akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedRunning.scala

```diff
@@ -257,14 +258,6 @@ private[akka] object EventsourcedRunning {
         event.getClass.getName, seqNr, setup.persistenceId, cause.getMessage)
     }
 
-    private def onPersistFailureThenStop(cause: Throwable, event: Any, seqNr: Long): Behavior[InternalProtocol] = {
-      setup.log.error(cause, "Failed to persist event type [{}] with sequence number [{}] for persistenceId [{}].",
-        event.getClass.getName, seqNr, setup.persistenceId)
-
-      // FIXME see #24479 for reconsidering the stopping behaviour
-      Behaviors.stopped
-    }
-
   }
 
```

Comment:
In EventSourcedReplayingEvents and EventsourcedReplayingSnapshot there are also `onRecoveryFailure`. I think we can treat those as a `PersistFailedException`, i.e. the backoff would be used for such failures also.

Reply:
:+1: